setup.py

```diff
@@ -13,7 +13,7 @@
 
 setup(
     name='awsprocesscreds',
-    version='0.0.1',
+    version='0.0.2',
```

Comment:
Do we want to pull the version from `__init__.py`? It is pretty easy to forget to update the version in both places. Probably can just copy code from our previous projects.